src/gazebo_motor_model.cpp

```diff
@@ -233,7 +243,7 @@ void GazeboMotorModel::UpdateForcesAndMoments() {
 #else
   ignition::math::Pose3d pose_difference = ignitionFromGazeboMath(link_->GetWorldCoGPose() - parent_links.at(0)->GetWorldCoGPose());
 #endif
-  ignition::math::Vector3d drag_torque(0, 0, -turning_direction_ * force * moment_constant_);
+  ignition::math::Vector3d drag_torque(0, 0, -turning_dir * force * moment_constant_);
```

Comment:
My main concern with making the reversible parameter part of the turning direction, is that the reference rotor velocity range are not explicitly defined. (It can range from 0~1 or -1~+1 depending on the `MAV_TYPE`( https://github.com/PX4/Firmware/blob/master/src/modules/simulator/simulator_mavlink.cpp#L133 ). This will add an additional layer of confusion when something is not working as intended.

Reply:
I've set MAV_TYPE to 7 (Airship, controlled) only and by sticking to -1..1, that should certainly help.